filters/tee/tee.go

```diff
@@ -88,40 +88,55 @@ func (r *tee) Response(filters.FilterContext) {}
 
 // Request is copied and then modified to adopt changes in new backend
 func (r *tee) Request(fc filters.FilterContext) {
-	copyOfRequest := cloneRequest(r, fc.Request())
+	req := fc.Request()
+	copyOfRequest, tr, err := cloneRequest(r, req)
+	if err != nil {
+		log.Warn("tee: error while cloning the tee request", err)
+		return
+	}
+
+	req.Body = tr
+
 	go func() {
-		rsp, err := r.client.Do(&copyOfRequest)
+		rsp, err := r.client.Do(copyOfRequest)
 		if err != nil {
-			log.Warn("error while tee request", err)
-		}
-		if err == nil {
-			defer rsp.Body.Close()
+			log.Warn("tee: error while tee request", err)
+			return
 		}
+
+		defer rsp.Body.Close()
```

Comment:
defer: true, fixing

ioutil.Discard: how to do it?
as i understand we're on the client side in this case, and we'll receive an *http.Response